VCFv4.3.tex

```diff
@@ -253,6 +253,17 @@ \subsubsection{Pedigree field format}
 ##pedigreeDB=URL
 \end{verbatim}
 
+
+\subsubsection{Reference block checkpoint}
```

Comment:
This should be placed **after** the final sentence (“See [PedigreeInDetail] for details”) of the preceding section.